include/deal.II/numerics/matrix_tools.h

```diff
@@ -68,6 +69,8 @@ namespace PETScWrappers
 #endif
 
 #ifdef DEAL_II_WITH_TRILINOS
+// TODO: move this to a stand-alone header and include from there?
+#  define TrilinosScalar double
 namespace TrilinosWrappers
```

Comment:
then i would prefer to keep this extra `define` here so that all functions are consistently defined. Extra define will be cleaned up together with others when Tpetra is here.